shuup/admin/templates/shuup/admin/products/_edit_media_form.jinja

```diff
@@ -90,17 +90,18 @@
 {%- set name = _("Product Images") if is_image_form else _("Product Media") %}
 
 {% call content_block(name, "fa-camera") %}
-    {{ media_form.management_form }}
-    {% for f in media_form %}
-        {{ render_media_form(f, media_form, loop.index, is_image_form) }}
-    {% endfor %}
-    <div class="hide" id="placeholder-panel">
-        {{ render_media_form(media_form.empty_form, media_form, "__prefix_name__", is_image_form) }}
+    {%- set target_panels = "product-images-section-wrapper" if is_image_form else "product-media-section-wrapper" %}
```

Comment:
Instead of wrapping these content_blocks could you pass the untranslated id for content_block. I think it's bad idea that the contect block uses the title as id.

Also using content_block should work without wrapping everything to extra div.